### ISSUE TITLE
vscode: UI never comes up Warning: 'ozone-platform' is not in the list of known options, but still passed to Electron/Chromium.

### ISSUE THREAD

**Issue by kjeremy** [NixOS/nixpkgs#172597]
### Describe the bug
I am trying to run `code` in a directory and the UI never comes up.  I see the following on the console

```
Warning: 'ozone-platform' is not in the list of known options, but still passed to Electron/Chromium.
```

### Steps To Reproduce
1. Run `vscode-fhs` with `code`
2. Observe the above warning
3. Observe that the UI never comes up

### Expected behavior
VSCode runs

### Screenshots
![image](https://user-images.githubusercontent.com/4325700/167930674-d972a12d-e4be-4d63-9d82-c07303b2e8ea.png)

### Additional context
This is on Wayland

### Notify maintainers

@eadwu 
@Synthetica9 
@maxeaubrey 
@bobby285271 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.37, NixOS, 22.05 (Quokka), 22.05.20220510.f419dc5`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.9.0pre20220505_f4102de`
 - channels(root): `"nixos"`
 - channels(jkolb): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by Garmelon** [comment]
Sounds like #172450, which should be solved by #172320

**Comment by ncfavier** [comment]
Closing as duplicate and solved.
